docs/content/status.mdx

```diff
@@ -18,20 +18,20 @@ import {
 <NetworkStatus
   networkName={MAINNET}
   networkVersion="9"
-  versionUrl=""
+  versionUrl="https://docs.onflow.org/node-operation/past-sporks/#mainnet-9"
   nextSporkDate="June 23, 2021"
 />
 <NetworkStatus
   networkName={TESTNET}
-  networkVersion="9"
-  versionUrl=""
+  networkVersion="Devne23"
```

Comment:
Why is the network called "Devne" and not "Devnet"?

Looks like there was a typo in 22 and then it was just copied?
https://docs.onflow.org/node-operation/past-sporks/#devne22

Reply:
Not sure how that got in there. 
This is what Vishal has written.

Reply:
Yup looks like it. I'll make another PR to fix this.

Reply:
Thanks Mack!